controllers/datadogagent/utils_test.go

```diff
@@ -183,6 +187,7 @@ func Test_getVolumeMountsForSecurityAgent(t *testing.T) {
 			dda:  testutils.NewDatadogAgent("foo", "bar", "datadog/agent:7", &testutils.NewDatadogAgentOptions{SecuritySpec: securityRuntime}),
 			want: []corev1.VolumeMount{
 				v1.VolumeMount{Name: "logdatadog", ReadOnly: false, MountPath: "/var/log/datadog"},
+				v1.VolumeMount{Name: "datadog-agent-auth", ReadOnly: true, MountPath: "/etc/datadog-agent/auth"},
```

Comment:
/etc/os-release should also be mounted to /host/etc/os-release

Reply:
what should be the trigger? runtine-security enabled?

Reply:
PR updated